workbench-core/example/infrastructure/src/index.ts

```diff
@@ -34,5 +34,5 @@ new WorkbenchAppRegistry(exampleStack, 'ExampleStack', {
   destroy: true,
   appInsights: true
 });
-
 Aspects.of(app).add(new AwsSolutionsChecks({ verbose: true }));
+app.synth();
```

Comment:
we don't take care of this in a separate synth step?

Reply:
we do not, I deployed a new example stack and it broke because this was not synth